setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = iohub
-author = Computational Microscopy Platform, CZ Biohub
-author_email = shalin.mehta@czbiohub.org
+author = CZ Biohub and iohub contributors
+author_email = iohub@czbiohub.org
```

Comment:
is the email ready-to-use already @ziw-liu ? could you send a test email to make sure.